Appraisals

```diff
@@ -1,29 +1,39 @@
-appraise "3.2" do
+appraise "3.2.awsv1" do
+  gem "rails", "~> 3.2.0"
+  gem "aws-sdk", "~> 1.5"
+end
+
+appraise "3.2.awsv2" do
   gem "rails", "~> 3.2.0"
   gem "aws-sdk", "~> 2.0"
 end
 
-appraise "4.1" do
+appraise "4.1.awsv1" do
   gem "rails", "~> 4.1.0"
-  gem "aws-sdk", "~> 2.0"
+  gem "aws-sdk", "~> 1.5"
 end
 
-appraise "4.2" do
-  gem "rails", "~> 4.2.0"
+appraise "4.1.awsv2" do
+  gem "rails", "~> 4.1.0"
   gem "aws-sdk", "~> 2.0"
 end
 
-appraise "3.2.awsv1" do
-  gem "rails", "~> 3.2.0"
+appraise "4.2.awsv1" do
+  gem "rails", "~> 4.2.0"
   gem "aws-sdk", "~> 1.5"
 end
 
-appraise "4.1.awsv1" do
-  gem "rails", "~> 4.1.0"
-  gem "aws-sdk", "~> 1.5"
+appraise "4.2.awsv2" do
+  gem "rails", "~> 4.2.0"
+  gem "aws-sdk", "~> 2.0"
 end
 
-appraise "4.2.awsv1" do
-  gem "rails", "~> 4.2.0"
+appraise "master.awsv1" do
+  gem 'rails', github: "rails/rails"
   gem "aws-sdk", "~> 1.5"
 end
```

Comment:
Not sure about v1 support (this would be another big breaking change deprecation), but I can look into this.